motoko/send_http_get/Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: all
+all: build
+
+.PHONY: node_modules
+.SILENT: node_modules
+node_modules:
+	npm install
+
+.PHONY: build
+.SILENT: build
+build: node_modules
+	dfx canister create send_http_get_backend
+	dfx build
+
+.PHONY: install
+.SILENT: install
+install: build
+	dfx canister install send_http_get_backend --mode reinstall --yes
+
+
+.PHONY: upgrade
+.SILENT: upgrade
+upgrade: build
+	dfx canister install send_http_get_backend --mode=upgrade
+
+.PHONY: test
+.SILENT: test
+test: install
+	dfx canister call send_http_get_backend get_icp_usd_exchange \
+		| grep '[[1682978460,5.714,5.718,5.714,5.714,243.5678]]' && echo 'PASS'
```

Comment:
In a regular expression:
- `[ab.c]` means match any of `a`, `b`, or `c`, or '.,'
- `.` outside of square brackets means match any character
So as written, this grep expression means something like:
`string contains any of 0 1 2 3 4 5 6 7 8 9 . followed by ]`
```
$ echo "5]" \                                             
                | grep '[[1682978460,5.714,5.718,5.714,5.714,243.5678]]' && echo 'PASS'
5]
PASS
$ echo ".]" \
                | grep '[[1682978460,5.714,5.718,5.714,5.714,243.5678]]' && echo 'PASS'
.]
PASS

```
If you are looking for single brackets around these values:
```suggestion
		| grep '\[1682978460,5\.714,5\.718,5\.714,5\.714,243\.5678\]' && echo 'PASS'
```

If you are looking for double brackets around these values:
```suggestion
		| grep '\[\[1682978460,5\.714,5\.718,5\.714,5\.714,243\.5678\]\]' && echo 'PASS'
```



```
$ echo "[[1682978460,5.714,5x718,5.714,5.714,243.5678]]" \
                | grep '\[1682978460,5\.714,5\.718,5\.714,5\.714,243\.5678\]' && echo 'PASS'
$ echo "[[1682978460,5.714,5.718,5.714,5.714,243.5678]]" \ 
                | grep '\[1682978460,5\.714,5\.718,5\.714,5\.714,243\.5678\]' && echo 'PASS'
[[1682978460,5.714,5.718,5.714,5.714,243.5678]]
PASS
```